dist/restore/index.js

```diff
@@ -1,22 +1,14 @@
 // GCS Cache Main Action - Restore caches
 
 // *** Imports ***
-// Node.js dependencies
-const fs = require('fs');
-const path = require('path');
-const md5 = require('md5');
 // GitHub Actions dependencies
 const core = require('@actions/core');
 const exec = require('@actions/exec');
 // Google Cloud client library
 // https://github.com/googleapis/nodejs-storage
 const {Storage} = require('@google-cloud/storage');
-
-// *** File compression configuration ***
-const temporaryDirectoryPrefix = 'temp';
-const compressionCommand = 'tar -zcf';
-const expandCommand = 'tar -zxf';
-const compressedFileExtension = '.tar.gz';
+// Module with GCS Cache Action's global functions
+const globalFunctions = require('../module');
```

Comment:
rename to sharedFunctions, OR directly import needed function via :
```suggestion
const {
  fileNameForCache,
 } = require('../module');
```